### ISSUE TITLE
🖊️⚙️ Implement Input and Validation for Max Base Fee and Priority Fee in Custom TX Parameters

### ISSUE THREAD

**Issue by vkjr** [status-im/status-react#21023]
# Feature

Implement max fees flow: https://www.figma.com/design/xLs1KYmF4e6WwRTZVJKeUK/Wallet?node-id=17671-228142&m=dev
Implement priority fee flow: https://www.figma.com/design/xLs1KYmF4e6WwRTZVJKeUK/Wallet?node-id=17671-229387&t=BUk6LX8HGnOk60xD-4

## TBD
- where to get current limits?

## Acceptance Criteria

- [ ] There should be error messages if fees are incorrect
- [ ] There should be a help section as shown in designs

**Comment by churik** [comment]
I do not think that it is feasible to include this feature to  2.32, postponing due to limited capacity in order to keep release scope clear

cc @shivekkhurana
